applications/openshift/authentication/ocp_idp_no_htpasswd/policy/stig/shared.yml

```diff
@@ -0,0 +1,123 @@
+srg_requirement: |-
+  {{{ full_name }}} must use a centralized user management solution to
+  support account management functions.
+
+vuldiscussion: |-
+  Enterprise environments make application account management challenging
+  and complex. A manual process for account management functions adds the
+  risk of a potential oversight or other error.
+
+checktext: |-
+  Run the following command to list the identity providers configured:
```

Comment:
should we mention there should no be HTPasswd here?

Reply:
yes, this one was wrong, thanks